framework/src/modules/chain/blocks/utils.js

```diff
@@ -96,7 +96,7 @@ const parseStorageObjToLegacyObj = block => {
 
 /**
  * Generates a list of full blocks for another node upon sync request from that node, see: modules.transport.internal.blocks.
- * NOTE: changing the original method loadBlocksData() could potentially change behaviour (popLastBlock() uses it for instance)
+ * NOTE: changing the original method loadBlocksDataWS() could potentially change behaviour (popLastBlock() uses it for instance)
```

Comment:
Probably name should be changed back to `loadBlocksData`

Reply:
I reckon it's fine to keep `loadBlocksDataWS` as it's only used by Network module but if we want to change the name I would suggest changing it for something more descriptive.

Reply:
yes true, it's only used for syncing.